src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Area {

```

Comment:
<img width="905" alt="image" src="https://user-images.githubusercontent.com/68428976/181244592-9101819d-24e5-4c8a-8838-751bb5bde463.png">
What about part of drawing figure? We should implement some Drawable interface with method `void draw()` and override it is all figure classes